src/Resources/Locales/en.json

```diff
@@ -1392,11 +1395,20 @@
                 "messages": {
                     "noColor": "Not set",
                     "loading": "Loading historized data...",
-                    "noData": "No historized data found in selected time frame.",
+                    "noData": "No historized data found in the selected time range.",
```

Comment:
can we remove `historized`? `No data found for the selected time range`.
This seems like it should/could match up with the message above (in the loc file) for `viewer.chart.messages.noData`

Reply:
after content pass with Azima and Riley it is "No numeric historized data found in the selected time range" for chart, "No historized data found in the selected time range" for Table. I think it is good to mentioned historized since this is timeseries data stored in Kusto. What do you think @azimamansuri @rinisbet , I can remove that part if you all agree.